src/TranscriptionOpt/TranscriptionOpt.jl

```diff
@@ -2,8 +2,7 @@ module TranscriptionOpt
 
 # Import the necessary packages.
 import JuMP
-import MathOptInterface
-const MOI = MathOptInterface
+const MOI = JuMP.MOI
```

Comment:
It's surprising that MOI doesn't get pulled in by `using JuMP`. Anyway, this seems to work.

Reply:
Here I just use `import Jump` which I wouldn't think pulls in any of its exported objects. However, I realized this is redundant this `using ..InfiniteOpt` used below it pulls in JuMP and its exports anyway. #158 removes this redundancy.